package challenges.exceptions.where_to_catch;

public class MyException extends Exception {}
